### PR TITLE
[Enhancement][Lake] Alter tablet task report

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

### CODE REVIEW

**Comment by sduzh** [comment]
run all

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 15 / 17 (88.24%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/task/AlterReplicaTask.java|15|17|88.24%|[213, 214]|

File: fe/fe-core/src/main/java/com/starrocks/task/AlterReplicaTask.java [hunk 1]
```diff
@@ -178,6 +197,11 @@ public void handleFinishAlterTask() throws MetaNotFoundException {
             }
             Tablet tablet = index.getTablet(getTabletId());
             Preconditions.checkNotNull(tablet, getTabletId());
+            if (tbl.isLakeTable()) {
+                setFinished(getFailedTimes() == 0);
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/task/AlterReplicaTask.java:201]: Task should be finished if retry successfully.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 9 (77.78%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/task/AlterReplicaTask.java|7|9|77.78%|[201, 202]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 27 / 31 (87.10%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/task/AlterReplicaTask.java|27|31|87.10%|[203, 212, 213, 214]|

File: fe/fe-core/src/main/java/com/starrocks/task/AlterReplicaTask.java (hunk 1 above)
Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/task/AlterReplicaTask.java]: updated

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 32 / 36 (88.89%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/task/AlterReplicaTask.java|27|31|87.10%|[197, 206, 207, 208]|
|:large_blue_circle:|com/starrocks/alter/SchemaChangeJobV2.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/alter/RollupJobV2.java|3|3|100.00%|[]|

**Review by wyb — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by sduzh** [comment]
run starrocks_admit_test

**Pull request merged**